src/m365/base/ContextCommand.ts

```diff
@@ -0,0 +1,40 @@
+import * as fs from 'fs';
+import AnonymousCommand from './AnonymousCommand';
+import { Logger } from '../../cli/Logger';
+import { M365RcJson } from './M365RcJson';
+
+export default abstract class ContextCommand extends AnonymousCommand {
+  public removeContextInfo(logger: Logger): void {
```

Comment:
Removing context info is specific to the remove command, so we should move it to the command class

Reply:
I have left the file 'ContextCommand.ts' in this pr despite it being an empty class. Because context remove command inherits from this class. Thus, the PR is not broken.

Should be merged with the context init pr. Let me know if you would like to see it otherwise.